server/common/utils/index.js

```diff
@@ -56,22 +56,56 @@ const responseWithItems = (userId, list) => {
   const { items } = list;
 
   return _map(items, item => {
-    const { voterIds, ...rest } = item.toObject();
+    const {
+      _id,
+      authorId,
+      authorName,
+      createdAt,
+      description,
+      isOrdered,
+      link,
+      name,
+      voterIds
+    } = item.toObject();
```

Comment:
why all these changes (replacing ... operator)?

Reply:
I changed these methods back. I used lean() method in controllers to receive clean data object without any mongo's wrapper and the spread operator can be used safely.